thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/components/PercentageChangeRuleDetector.java

```diff
@@ -76,16 +80,34 @@
 
   @Override
   public DetectionResult runDetection(Interval window, String metricUrn) {
+    long startMillis = window.getStartMillis();
+
+    // align start day to the user specified week start
+    if (Objects.nonNull(this.weekStart)) {
```

Comment:
Why use Objects.notNull instead of this.weekStart == null?

Reply:
They should be equivalent in functionality. This is more of java 8 style.